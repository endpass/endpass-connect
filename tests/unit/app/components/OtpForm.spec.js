import { shallowMount, mount } from '@vue/test-utils';
import OtpForm from '@/components/OtpForm.vue';

describe('OtpForm', () => {
  describe('render', () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallowMount(OtpForm, {
        propsData: {},
      });
    });

    it('should correctly render OtpForm component', () => {
      expect(wrapper.name()).toBe('OtpForm');
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
