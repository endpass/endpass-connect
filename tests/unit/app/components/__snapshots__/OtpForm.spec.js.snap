// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtpForm render should correctly render OtpForm component 1`] = `
<form>
  <form-field-stub>
    <message-stub data-test="form-message">Enter the 6 digit code generated by your authenticator app.</message-stub>
  </form-field-stub>
  <!---->
  <form-field-stub>
    <v-input-stub placeholder="Enter received code..." name="code" value="" invalid="true" autofocus="true" type="number" data-test="email-input"></v-input-stub>
  </form-field-stub>
  <form-controls-stub>
    <v-button-stub disabled="true" type="primary" submit="true" data-test="submit-button">
      Submit code
    </v-button-stub>
    <v-button-stub data-test="cancel-button">Close</v-button-stub>
  </form-controls-stub>
</form>
`;
